packages/blocks/src/components/button-spinner/style.scss

```diff
@@ -0,0 +1,12 @@
+.crowdsignal-forms-button-spinner {
```

Comment:
Hmm... I believe we've made the decision to use styled components/`@emotion` for our styles.  
Compatibility stylesheets/editor being the sole exception. Would be nice to keep it consistent.

I guess the benefit would be everything is contained within a single file and we have easy control of where we inject the generated classes. This is not much of an issue now but we will eventually want to move on from our project-renderer/dashboard combo and offer additional apps and embeds where this could impact their practicality.